### PR TITLE
Created new documentation for the new Ambient Weather platform compon…

### PR DESCRIPTION
**Pull request by driveawedge**
…ent.

**Description:**

Added documentation for new Ambient weather component.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19178

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/weather.ambient.markdown [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Ambient"
+description: "Instructions on how to integrate Ambient weather network within Home Assistant."
```
Comment (frenck) [inline source/_components/weather.ambient.markdown]:
```suggestion
description: "Instructions on how to integrate Ambient Weather Network within Home Assistant."
```

File: source/_components/weather.ambient.markdown [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+title: "Ambient"
```
Comment (frenck) [inline source/_components/weather.ambient.markdown]:
```suggestion
title: "Ambient Weather"
```

File: source/_components/weather.ambient.markdown [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+description: "Instructions on how to integrate Ambient weather network within Home Assistant."
+date: 2018-11-24 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: ambient-weather-logo.png
+ha_category: Weather
+ha_release: 0.32
```
Comment (frenck) [inline source/_components/weather.ambient.markdown]: This seems a little low... 😉

File: source/_components/weather.ambient.markdown [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+ha_iot_class: "Cloud Polling"
+---
+
+The `ambient` weather platform uses [Ambient Weather](https://ambientweather.net/) as a source for information from a local weather station. To use this platform you must have an Ambient weather station installed at your location and the station must support sending weather data over the Internet to Ambient.
+
+## {% linkable_title Configuration %}
+
+You need an API key which is free but requires a [registration](https://ambientweather.net/).
+
+To add ambientweather to your installation, add the following to your `configuration.yaml` file:
```
Comment (frenck) [inline source/_components/weather.ambient.markdown]:
```suggestion
To add Ambient Weather to your installation, add the following to your `configuration.yaml` file:
```

File: source/_components/weather.ambient.markdown [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+```yaml
+# Example configuration.yaml entry
+weather:
+  - platform: ambient
+    api_key: YOUR_API_KEY
+```
+
+{% configuration %}
+api_key:
+  description: Your API key for [Ambient](https://ambientweather.net/).
```
Comment (frenck) [inline source/_components/weather.ambient.markdown]:
```suggestion
  description: Your API key for [Ambient Weather](https://ambientweather.net/).
```

File: source/_components/weather.ambient.markdown [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+comments: false
+sharing: true
+footer: true
+logo: ambient-weather-logo.png
+ha_category: Weather
+ha_release: 0.32
+ha_iot_class: "Cloud Polling"
+---
+
+The `ambient` weather platform uses [Ambient Weather](https://ambientweather.net/) as a source for information from a local weather station. To use this platform you must have an Ambient weather station installed at your location and the station must support sending weather data over the Internet to Ambient.
```
Comment (frenck) [inline source/_components/weather.ambient.markdown]:
```suggestion
The `ambient` weather platform uses [Ambient Weather](https://ambientweather.net/) as a source for information from a local weather station. To use this platform you must have an Ambient Weather station installed at your location and the station must support sending weather data over the Internet to Ambient Weather.
```

**Review by frenck — CHANGES_REQUESTED** [review]
@driveawedge thanks for this PR! I've reviewed it, and left you some comments, could you please take a look? 👍

**Comment by driveawedge** [comment]
Tried to make the changes, wound up creating another disaster of my git branch. I'm going to just give up trying to contribute because I can't figure out git.

**Pull request closed without merging**